src/tests/readme.md

```diff
@@ -0,0 +1,158 @@
+#Implementing interactions tests using Storybook & Vibe
+For implementing more complex e2e tests in the storybook environment, we recommend using storybook interaction tests combined with the tools exported by our repository.
+For example, interaction tests can be a more straightforward tool for testing keyboard navigation support flow on your composites or pages, happy path flows, or reproducing error or empty complex scenarios.
+
+##Implement interactions tests for stories by using interactions suites
+Interactions tests are functions that can interact with your story elements.
+You can see an example of connecting a test function to a story below:
+````
+// LoginForm.stories.ts|tsx
+
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { within, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+import { LoginForm } from './LoginForm';
+
+export default {
+  title: 'Form',
+  component: LoginForm,
+} as ComponentMeta<typeof LoginForm>;
+
+const Template: ComponentStory<typeof LoginForm> = (args) => <LoginForm {...args} />;
+export const EmptyForm = Template.bind({});
+
+/*
+* See https://storybook.js.org/docs/react/writing-stories/play-function#working-with-the-canvas
+* to learn more about using the canvasElement to query the DOM
+*/
+export const FilledForm = Template.bind({});
+FilledForm.play = async ({ canvasElement }) => {
```

Comment:
Is it possible to write these tests in a separate file? Maybe we should consider it - just not to create a mess

Reply:
It's just an example of how to use the play function and write a test, as a standard I agree.